src/main/java/org/jabref/gui/entryeditor/EntryEditor.fxml

```diff
@@ -39,6 +39,14 @@
                     <Tooltip text="%Change entry type"/>
                 </tooltip>
             </Button>
+            <Button fx:id="generateKeyButton" styleClass="flatButton,narrow">
```

Comment:
Please set the on-button-click-action via fxml as below (`onAction="#navigateToPreviousEntry"`).